kahuna/public/js/preview/image.html

```diff
@@ -173,6 +174,14 @@
             </div>
         </span>
 
+        <a ng-if="ctrl.image.data.identifiers['original-media-id']"
+           class="bottom-bar__action preview__has-original-media-id"
+           title="{{'this image has has an Original Media ID\ni.e. has been modified'}}"
```

Comment:
The nittiest of nits: I would capitalise the tooltip. And also, Guardian style is weird about them and [requires](https://www.theguardian.com/guardian-observer-style-guide-i) `ie` (no fullstops at all). Not sure how Grid writes them elsewhere, but, for ease and to be nice to our partners in crime, I would just avoid? Maybe just `This image has an Original Media ID: has been modified`? (but I’m no English speaker!)

Reply:
ok, addressed, it now reads `This image has an Original Media ID (it has been modified)`